conjure-java-core/src/integrationInput/java/com/palantir/product/PrimitiveOptionalsExample.java

```diff
@@ -213,7 +213,7 @@ public Builder from(PrimitiveOptionalsExample other) {
             return this;
         }
 
-        @JsonSetter(value = "num", nulls = Nulls.SKIP)
+        @JsonSetter(value = "num", nulls = Nulls.SKIP, contentNulls = Nulls.FAIL)
         public Builder num(OptionalDouble num) {
```

Comment:
What does `, contentNulls = Nulls.FAIL` do here? I thought that was for nested values in collections.